### DISCUSSION TITLE
Is it possible to run sockets concurrently

### DISCUSSION THREAD

**Discussion by AliAlican** [spatie/fork#16] [Q&A]
Can i use this package to connect to multiple sockets at the same time?

**Comment by brendt** [comment]
I think so, you'd need to do some tinkering with it.

**Reply by AliAlican** [reply]
Do you mean tinkering on my implementation, or will i need to override some methods?

**Reply by brendt** [reply]
I'm not sure, that's what tinkering is for 😊
